src/value.h

```diff
@@ -12,19 +12,17 @@
 #include "redisearch.h"
 #include "rmalloc.h"
 #include "query_error.h"
-#include "reply.h"
 
 #include "util/fnv.h"
 
-#include "rmutil/args.h"
 #include "rmutil/rm_assert.h"
 #include "hiredis/sds.h"
 
+#include <assert.h>
```

Comment:
Redundant (use `RS_ASSERT` from `"rmutil/rm_assert.h"`)

Reply:
Fixed in ddbb2986dc86c67e8e83d6cc71c99dce5ca5c7bc